### PR TITLE
[1] Add builds.json files for some projects

### PR DESCRIPTION
**Pull request by mchindri**

### CODE REVIEW

**Review by dbogdan — APPROVED** [review]

File: projects/ad7616-sdz/builds.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+      "xilinx": {
+        "demo": {
+              "flags" : "",
+              "hardware" : ["ad7616_sdz_zed", "ad7616_sdz_zc706"]
+        }
+      }
+}
```
Comment (dbogdan) [inline projects/ad7616-sdz/builds.json:8]: I would fix this warning for all the files.

Comment (mchindri) [reply projects/ad7616-sdz/builds.json]: done

**Comment by mchindri** [comment]
v2: 
 - Add new line at end of files

**Comment by mchindri** [comment]
v3: 
 - Add some fixes.
 - Add one more builds.json

**Comment by mchindri** [comment]
v4:
 - Add one more project to build
 - Add hardware option for build_projects.py

**Review by buha — APPROVED** [review]

**Comment by mchindri** [comment]
v5: 
 - Add more small fixes to builds_projects.json

**Pull request merged**